codeview/src/main/java/io/github/kbiakov/codeview/adapters/CodeWithNotesAdapter.kt

```diff
@@ -11,23 +13,27 @@ import io.github.kbiakov.codeview.views.LineNoteView
  *
  * @author Kirill Biakov
  */
-class CodeWithNotesAdapter : AbstractCodeAdapter<String> {
+open class CodeWithNotesAdapter : AbstractCodeAdapter<String> {
```

Comment:
You are used Kotlin ```open class```, but I not found related open methods and it's final class which is not necessary to inherit. Can you please explain your choice?